src/main/asciidoc/groovy/index.adoc

```diff
@@ -0,0 +1,41 @@
+= Vert.x-tcp-eventbus-bridge
+:toc: left
+
+Vert.x-tcp-eventbus-bridge is a TCP bridge to Vert.x EventBus.
+
+To use this project, add the following
+dependency to the _dependencies_ section of your build descriptor:
+
+* Maven (in your `pom.xml`):
+
+[source,xml,subs="+attributes"]
+----
+<dependency>
+  <groupId>{maven-groupId}</groupId>
```

Comment:
wrong syntax (has been changed in a hurry just before the release. Use ${maven.groupId}, ${maven.artifactId} and ${maven.version}